engines/bastion_katello/app/assets/javascripts/bastion_katello/ostree-branches/details/ostree-branches-details.controller.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.ostree-branches.controller:OstreeBranchesDetailsController
+ *
+ * @requires $scope
+ * @requires OstreeBranch
+ *
+ * @description
+ *   Provides the functionality for the Ostree Branch action pane.
+ */
+angular.module('Bastion.ostree-branches').controller('OstreeBranchesDetailsController', ['$scope', 'OstreeBranch',
+    function ($scope, OstreeBranch) {
+        if ($scope.branch) {
+            $scope.panel = {loading: false};
+        } else {
+            $scope.panel = {loading: true};
+        }
+
+        $scope.branch = OstreeBranch.get({id: $scope.$stateParams.branchId}, function () {
+            $scope.panel.loading = false;
+        });
```

Comment:
Meant to mention above that you need to follow the example [here ](https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/activation-key-details.controller.js#L33-L39) in order to remove the loading spinner on an error and display the error message.